spec/javascripts/app/views/preview_post_view_spec.js

```diff
@@ -0,0 +1,95 @@
+describe("app.views.PreviewPost", function() {
+  beforeEach(function() {
+    this.model = new app.models.Post({
+      "id": 0,
+      "text": "text",
+      "public": "public",
+      "created_at": new Date(),
+      "interacted_at": new Date(),
+      "author": {},
+      "mentioned_people": [],
+      "photos": [],
+      "frame_name": "status",
+      "title": "",
+      "location": ["", ""],
+      "interactions": {"likes": [], "reshares": [], "comments_count": 0, "likes_count": 0, "reshares_count": 0},
+      "poll": undefined
+    });
```

Comment:
https://github.com/diaspora/diaspora/blob/develop/spec/javascripts/jasmine_helpers/factory.js#L184

`spec/javascripts/app/views/poll_view_spec.js:4:    this.view = new app.views.Poll({ model: factory.postWithPoll()});`